pkg/operator/controllers/workaround/systemreserved.go

```diff
@@ -60,13 +61,13 @@ func (sr *systemreserved) IsRequired(clusterVersion *version.Version) bool {
 }
 
 func (sr *systemreserved) kubeletConfig() (*mcv1.KubeletConfig, error) {
+	customEvictionHard := v1beta1.DefaultEvictionHard
```

Comment:
Dont want to vendor all this just so we can add 4 lines of code. No go :) Just copy values out